server/config/options.go

```diff
@@ -51,19 +54,25 @@ type CdcConfig struct {
 }
 
 type TracingConfig struct {
-	Enabled           bool
-	WithUDS           string
-	WithAgentAddr     string
-	WithDogStatsdAddr string
+	Enabled           bool   `mapstructure:"enabled" yaml:"enabled" json:"enabled"`
```

Comment:
What will be the use of mapstructure tag here?

Reply:
My understanding is that this enables the key to be used as an environment variable with `spf13/cobra` and `AutoEnv()`.